content/tutorials/get-started/set-up-secure-signing.md

```diff
@@ -62,21 +62,48 @@ As always, follow industry-standard practices for securing your machines, such a
 
 {{ include_svg("img/secure-signing-client-library.svg", "Diagram of using a client library with local signing") }}
 
-This configuration uses a client library in the programming language you are using to sign transactions locally. You need a client library for the programming language you use. Ripple publishes the following client libraries that can sign transactions for the XRP Ledger locally:
+This configuration uses a client library in the programming language you are using to sign transactions locally. You need a client library for the programming language you use. The following client libraries can sign transactions for the XRP Ledger locally:
 
-- **RippleAPI (ripple-lib) for JavaScript**
-    - [Setup](get-started-with-rippleapi-for-javascript.html)
-    - [API Reference](rippleapi-reference.html)
-- **Signing Library for C++** (included with `rippled`)
-    - [Documentation](https://github.com/ripple/rippled/tree/develop/Builds/linux#signing-library)
+- **Python**
+    - `xrpl-py`
+        - [Repo](https://github.com/XRPLF/xrpl-py)
+        - [Get Started](get-started-using-python.html)
+        - [API Reference](https://xrpl-py.readthedocs.io/en/latest/)
+- **JavaScript**
+    - `ripple-lib`
+        - [Repo](https://github.com/ripple/ripple-lib)
+        - [Get Started](get-started-with-rippleapi-for-javascript.html)
+        - [API Reference](rippleapi-reference.html)
+- **C++** (included with `rippled`)
+    - `rippled` Signing Library
+        - [Get Started](https://github.com/ripple/rippled/tree/develop/Builds/linux#signing-library)
```

Comment:
This ends up taking up a lot of space when you build it. What do you think about turning it into a more compact table?

| Current (list) | Suggested (table) |
|---|---|
| ![2021-04-05_194228_535253151](https://user-images.githubusercontent.com/7515597/113652060-f314c080-9647-11eb-83b4-d9845e55b41d.png) | ![2021-04-05_194656_244096027](https://user-images.githubusercontent.com/7515597/113652094-0162dc80-9648-11eb-9d90-c7968691508f.png) |

```suggestion
| Language | Library Name | Get Started | API Reference | Source Code |
|----------|--------------|-------------|---------------|-------------|
| Python   | xrpl-py      | [Get Started](get-started-using-python.html) | [API Reference](https://xrpl-py.readthedocs.io/) | [Repo](https://github.com/XRPLF/xrpl-py) |
| JavaScript / TypeScript | ripple-lib | [Get Started](get-started-with-rippleapi-for-javascript.html) |  [API Reference](rippleapi-reference.html) | [Repo](https://github.com/ripple/ripple-lib) |
| C++      | `rippled` Signing Library | [Get Started](https://github.com/ripple/rippled/tree/develop/Builds/linux#signing-library) |  | (Part of [`rippled`](https://github.com/ripple/rippled/))
```

Maybe to allow multiple libraries in the same language later, we could put the language in the second column? I dunno.

(Also, ripple-lib can be used in TypeScript now, so I mentioned that.)